packages/core/src/util/config.ts

```diff
@@ -198,6 +198,10 @@ export class Config {
     return getEnvVarOrFail("FEATURE_FLAGS_TABLE_NAME");
   }
 
+  static getHeartbeatRateLimitTableName(): string {
```

Comment:
I suggest we name this something more generic, we can prob reuse this to any outbound calls we do (to different 3rd party services). Maybe something like outboundRateLimit, and add a TSDoc explaining what its for?